### ISSUE TITLE
feature request: allow user to change sizes of each window (like in RStudio)

### ISSUE THREAD

**Issue by hillarysanders** [yhat/rodeo#104]
Rodeo!
:dancer: + :horse_racing:

**Comment by sjobeek** [comment]
This would definitely make a big difference in usability. 

A hotkey to toggle the relative sizes of the console and editor window would be a decent short-term solution as well.

**Comment by glamp** [comment]
https://github.com/yhat/rodeo/tree/window-splitter

Styles need some love but the concept is there

**Comment by sjobeek** [comment]
What's the right approach to install and test this out?

I tried cloning the repo, switching to window-splitter branch, installing with `python setup.py build` and `python setup.py install`, then running via `rodeo . --host=0.0.0.0 --no-browser`, but I ran into the following exception I wasn't able to move past:

```
 _______      ___   ______   ________    ___
|_   __ \   .'   `.|_   _ `.|_   __  | .'   `.
  | |__) | /  .-.  \ | | `. \ | |_ \_|/  .-.  \
  |  __ /  | |   | | | |  | | |  _| _ | |   | |
 _| |  \ \_\  `-'  /_| |_.' /_| |__/ |\  `-'  /
|____| |___|`.___.'|______.'|________| `.___.'

''''''''''''''''''''''''''''''''''''''''''''''''''
  URL: http://localhost:5000/
  DIRECTORY: /root/work/rodeo/rodeo
''''''''''''''''''''''''''''''''''''''''''''''''''
[ERROR]: Exception on / [GET]
Traceback (most recent call last):
  File "/root/miniconda/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/root/miniconda/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/root/miniconda/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/root/miniconda/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/root/miniconda/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/root/miniconda/lib/python2.7/site-packages/rodeo-0.3.6-py2.7.egg/rodeo/rodeo.py", line 71, in home
    file_tree=file_tree, version=__version__)
  File "/root/miniconda/lib/python2.7/site-packages/flask/templating.py", line 128, in render_template
    context, ctx.app)
  File "/root/miniconda/lib/python2.7/site-packages/flask/templating.py", line 110, in _render
    rv = template.render(context)
  File "/root/miniconda/lib/python2.7/site-packages/jinja2/environment.py", line 969, in render
    return self.environment.handle_exception(exc_info, True)
  File "/root/miniconda/lib/python2.7/site-packages/jinja2/environment.py", line 742, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/root/miniconda/lib/python2.7/site-packages/rodeo-0.3.6-py2.7.egg/rodeo/templates/index.html", line 17, in top-level template code
    {% include 'partials/panes/nav.html' %}
  File "/root/miniconda/lib/python2.7/site-packages/flask/templating.py", line 64, in get_source
    raise TemplateNotFound(template)
TemplateNotFound: partials/panes/nav.html
```
nav.html is definitely present in the `/templates/partials/panes/` directory...  not sure whats going on  :/

**Comment by sjobeek** [comment]
Ahh, got it. Needed to add 
```
package_data={
        "rodeo": [
            ...
            "templates/partials/panes/*.html",
        ]
    },
```
in setup.py.


It looks great!  

Minor issues with extraneous vertical scroll-bars in the text/console area, but it's already very usable.

**Comment by glamp** [comment]
thanks! can you submit a PR?

**Comment by sjobeek** [comment]
Yup - see #106

**Comment by KristianHolsheimer** [comment]
It'd be really great to be able to resize viewports!

What's the current status of sjobeek's pull request? I just installed a fresh clone, but I wasn't able to resize anything yet..

**Comment by glamp** [comment]
it's released now :+1:

**Comment by hillarysanders** [comment]
:) nice! thanks

On Wed, Jun 10, 2015 at 5:55 AM, Greg Lamp <notifications@github.com> wrote:

> Closed #104 <https://github.com/yhat/rodeo/issues/104>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/yhat/rodeo/issues/104#event-327393722>.
>



-- 
- Hillary

**Comment by KristianHolsheimer** [comment]
I saw, thanks!
